powerfulseal/policy/action_clone.py

```diff
@@ -189,12 +189,31 @@ def update_labels(**kwargs):
     return True
 
   def mutate_traffic_control(self, body, spec):
-    """ Adds an init container with tc """
+    """ Adds an init container with tc setup """
     if body.spec.template.spec.init_containers is None:
       body.spec.template.spec.init_containers = []
+    setup_spec = spec.get("setup")
     body.spec.template.spec.init_containers.append(
       kubernetes.client.V1Container(
-        name="chaos"+str(1+len(body.spec.template.spec.init_containers)),
+        name="chaos-setup-"+str(1+len(body.spec.template.spec.init_containers)),
+        command=setup_spec.get("command", "/bin/sh"),
+        args=setup_spec.get("args", "tc show eth0 && tc qdisc add dev eth0 root handle 1: prio bands 2 priomap 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 && tc show eth0"),
```

Comment:
Hey, could you explain a little this? Perhaps detail in the comment?

Reply:
I'm going to remove it for a no-op debug log. It won't always be applicable and I'll set this is a helper tc wrapper